tools/pipelines/build-client.yml

```diff
@@ -32,6 +32,7 @@ trigger:
   branches:
     include:
     - main
+    - next
```

Comment:
how does this work for release/prerelease for npm? i assume we only want to allow pre-release from next. can be handled in a follow up

Reply:
prereleases from next makes sense, though not sure if we would also want prereleases of a main patch version.  will follow up on this